var/spack/repos/builtin/packages/halide/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Halide(CMakePackage, PythonExtension):
+    """Halide is a programming language designed to make it easier to write
+    high-performance image and array processing code on modern machines."""
+
+    homepage = "https://halide-lang.org/"
+    url = "https://github.com/halide/Halide/archive/refs/tags/v14.0.0.tar.gz"
+    git = "https://github.com/halide/Halide.git"
+    maintainers = ["wraith1995"]
+    version("14.0.0", sha256="f9fc9765217cbd10e3a3e3883a60fc8f2dbbeaac634b45c789577a8a87999a01")
+    version("main", branch="main")
+    variant(
+        "build_type",
+        default="Release",
+        description="The build type to build",
+        values=("Release", "Debug", "RelWithDebInfo"),
+    )
+    generator = "Ninja"
+    variant("python", default=False, description="Install python bindings")
+    variant("tutorials", default=False, description="Install the Halide Tutorials.")
+    variant("utils", default=False, description="Install the Halide Utilities.")
+    variant("tests", default=False, description="Build and Run Halide Tests and Apps.")
+    extends("python", when="+python")
+
+    _targets = " targets=arm,x86,nvptx,aarch64,hexagon,webassembly "
+    depends_on("cmake@3.24.3", type="build")
+    depends_on("ninja", type="build")
+    depends_on(
+        "llvm@14.0.0:14+clang+lld" + _targets + " build_type=Release",
+        type=("link", "run"),
+    )
+    depends_on("libjpeg", type=("build", "link", "run"))
+    depends_on("libpng", type=("build", "link", "run"))
+
+    depends_on("python@3.10:", type=("build", "link", "run"), when="+python")
+    depends_on("py-pybind11@2.6.2:", type="build", when="+python")
+    depends_on("py-setuptools@43:", type="build", when="+python")
+    depends_on("py-scikit-build", type="build", when="+python")
+    depends_on("py-wheel", type="build", when="+python")
+    depends_on("py-build", type="build", when="+python")
+
+    depends_on("py-imageio", type="run", when="+python")
+    depends_on("py-pillow", type="run", when="+python")
+    depends_on("py-scipy", type="run", when="+python")
+    depends_on("py-numpy@1.0.0:", type="run", when="+python")
+
+    def cmake_args(self):
+
+        spec = self.spec
+        llvmdir = spec["llvm"].prefix.lib + "/cmake/llvm"
```

Comment:
@trws @haampie is there a more robust way of finding this directory since lib may be lib64 on other systems?

Reply:
For what it is worth I copied this from: https://github.com/spack/spack/blob/fce95e2efb76da3c6ee549f2a9b4e257fb42ae50/var/spack/repos/builtin/packages/enzyme/package.py

Reply:
The short answer is `llvm-config --libdir`.  I don't remember off top of my head if we have a `.libdir` property for that on the llvm package, but we could.

Reply:
Looks like you can also get away with ` llvm-config --cmakedir`. I will add and resubmit soon.

Reply:
@trws Is this fine?